index.html

```diff
@@ -46,68 +46,160 @@ <h1>
+          like what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </h3>
+        <p class="top-connect">Let's connect</p>
+        <ul class="top-social-media">
+          <li class="flex-item">
+            <a href=""><img src="./img/github.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/Linkedin-icon.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/angellist.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""><img src="./img/twitter.svg" alt="" /></a>
+          </li>
+          <li class="flex-item">
+            <a href=""
+              ><img src="./img/medium.svg" alt="" class="flex-item"
+            /></a>
+          </li>
+        </ul>
+        <button type="button" class="normal-button">Get my resume</button>
+      </section>
+      <div class="skill-list">
+        <details class="frame">
+          <summary class="frame-summary">Languages</summary>
+          <div class="language-list">
+            <img src="./img/js-lang.svg" alt="" />
+            <img src="./img/html-lang.svg" alt="" />
+            <img src="./img/css-lang.svg" alt="" />
+          </div>
+        </details>
+        <details class="frame">
+          <summary class="frame-summary">Frameworks</summary>
+        </details>
+        <details class="frame">
+          <summary class="frame-summary">Skills</summary>
+        </details>
+      </div>
     </section>
-    <section class="card-works-1">
-      <img src="./img/Snapshoot-Portfolio-3.svg" alt="Snapshoot Portfolio" />
-      <h2 class="project-title">Multi-Post Stories</h2>
-      <ul class="frame-2">
-        <li class="client">Canopy</li>
-        <li><img src="./img/Counter.svg" alt="" /></li>
-        <li class="role">Back End Dev</li>
-        <li><img src="./img/Counter.svg" alt="" /></li>
-        <li class="year-label">2015</li>
-      </ul>
-      <p class="primary-text">
-        A daily selection of privately personalized reads; no accounts or
-        sign-ups required.
+    <section class="contact-me">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
       </p>
-      <ul class="tags">
-        <li class="tag">html</li>
-        <li class="tag">css</li>
-        <li class="tag">javascript</li>
-      </ul>
-      <button type="button">See Project</button>
+      <input type="text" value="Yeremias" />
+      <input type="text" value="natajayanj14@gmail.com" />
+      <textarea
+        name="message"
+        id="message"
+        cols="30"
+        rows="10"
+        placeholder="Write your message here"
+      ></textarea>
+      <button type="submit">Get In Touch</button>
```

Comment:
- [ ] Kindly let's make sure our design looks identical to Figma's one, looks like our button has a border can we fix that, please?

Your Implementation    |  Figma Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/87689547/148921592-a9ea73bf-a1fa-4040-b7fb-64325a94e4f4.png) | ![image](https://user-images.githubusercontent.com/87689547/148921542-dadeca00-1119-4467-8798-a9e3b2fcac7e.png)

Reply:
Done